dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HttpUtils.java

```diff
@@ -154,10 +154,10 @@ public static String getResponseContentString(HttpGet httpget, CloseableHttpClie
                     logger.warn("http entity is null");
                 }
             } else {
-                logger.error("http get:{} response status code is not 200!", response.getStatusLine().getStatusCode());
+                logger.error("http get: {} response status code is not 200!", response.getStatusLine().getStatusCode());
             }
-        } catch (IOException ioe) {
-            logger.error(ioe.getMessage(), ioe);
+        } catch (NullPointerException | IOException exception) {
+            logger.error(exception.getMessage(), exception);
```

Comment:
I would recommend to verify the parameters before the method is processed.

Reply:
Hi @zhuangchong ，updated，please check again.
Thanks for your suggestion.